src/librustdoc/clean/types.rs

```diff
@@ -641,6 +643,53 @@ impl Item {
             _ => false,
         }
     }
+
+    crate fn hir_id(&self) -> hir::HirId {
```

Comment:
It would be better to return Option here, or at least document that this panics for non-local items.

Reply:
Good point. I'll document then.